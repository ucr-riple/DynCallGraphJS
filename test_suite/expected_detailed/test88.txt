{
    'global (test88.js:14:16:14:29)': [
        'Monster (test88.js:1:1:3:2)'
    ],
    'global (test88.js:15:14:15:43)': [
        'Proxy (Native)(test88.js:15:14:15:43)'
    ],
    'system (Native)': [
        'set (test88.js:6:8:12:4)'
    ]
}