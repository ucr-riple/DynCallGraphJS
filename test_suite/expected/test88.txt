{
  '(test88.js:14:16:14:29)': [
    '(test88.js:1:1:3:2)'
  ],
  '(test88.js:15:14:15:43)': [ 'Proxy (Native)' ],
  'system (Native)': [
    '(test88.js:6:8:12:4)'
  ],
  '(test88.js:8:7:8:63)': [ 'log (Native)' ]
}